bugsnag-android-core/src/main/java/com/bugsnag/android/EventStore.java

```diff
@@ -61,7 +61,7 @@ public int compare(File lhs, File rhs) {
     }
 
     void flushOnLaunch() {
-        if (config.getLaunchCrashThresholdMs() != 0) {
+        if (config.getLaunchDurationMillis() != 0) {
```

Comment:
I assume this is just a like-for-like change and we're going to amend this behaviour in a future change on the project wrt to sending launch crashes synchronously being controlled by the new config option?

Reply:
Yes - this behaviour will be altered in future changesets.